src/libstd/panicking.rs

```diff
@@ -71,6 +73,7 @@ enum Hook {
 
 static HOOK_LOCK: RWLock = RWLock::new();
 static mut HOOK: Hook = Hook::Default;
+#[cfg(feature = "backtrace")]
```

Comment:
Could some of these imports move down below in the `#[cfg]`'d block to avoid the number of `#[cfg]` annotations?